templates/rabbitmq.config.erb

```diff
@@ -16,18 +19,16 @@
     {tcp_listeners, []},
 <%- end -%>
 <%- if @ssl -%>
-    <%- if @ssl_versions -%>
-      {ssl, [{versions, [<%= @ssl_versions.sort.map { |v| "'#{v}'" }.join(', ') %>]}]},
-    <%- end -%>
     {ssl_listeners, [<%= @ssl_port %>]},
-    {ssl_options, [<%- if @ssl_cacert != 'UNSET' -%>{cacertfile,"<%= @ssl_cacert %>"},<%- end -%>
-                    {certfile,"<%= @ssl_cert %>"},
-                    {keyfile,"<%= @ssl_key %>"},
-                    {verify,<%= @ssl_verify %>},
-                    {fail_if_no_peer_cert,<%= @ssl_fail_if_no_peer_cert %>}
-                    <%- if @ssl_versions -%>
-                      ,{ssl, [{versions, [<%= @ssl_versions.sort.map { |v| "'#{v}'" }.join(', ') %>]}]}
-                    <% end -%>]},
+    {ssl_options, [<%- if @ssl_cacert != 'UNSET' -%>{cacertfile,"<%= @ssl_cacert %>"},
+                   <% end %>{certfile,"<%= @ssl_cert %>"},
```

Comment:
We could keep it where it was but change `<%- end -%>` to `<% end %>`, then the template looks okay and the output is prettier.